arrocke/gloss-translation
# feat: next unapproved verse

## What has changed
- Created `Next Unapproved` button placed in the toolbar next to the verse reference. Pressing this button advances the user to the next verse with unapproved glosses. 
- Created the api path `GET /api/languages/${language}/verses/${verseId}/next-unapproved-verse`



## Connected Issues

closes #356

## QA Steps
1. From any verse, press the button. Confirm that the verse switched to does not have all word glosses approved. Confirm that all verses between the initial verse and the current verse are approved. 
2. Approve all glosses in the current verse
3. Go to the previous verse and press the `Next Unapproved` button. Confirm that the verse approved in step 2 was skipped.


## Post-Deployment
